libandroid/lib/src/main/java/com/mapbox/services/android/geocoder/ui/GeocoderAutoCompleteView.java

```diff
@@ -108,6 +108,19 @@ public void setType(String type) {
   }
 
   /**
+   * parameter limits results to a set of one or more countries
+   * <a href="https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2">ISO 3166 alpha 2</a> country codes,
+   * separated by commas.
+   *
+   * @param country <a href="https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2">ISO 3166 alpha 2</a>
+   *                country codes, separated by commas.
+   * @since 2.0.0
+   */
+  public void setCountry(String country) {
```

Comment:
We might want to offer a varargs method here for convenience so that the comma separated format is abstracted:

```
public void setCountries(String... countries)
```